src/public/js/_.js

```diff
@@ -323,7 +323,7 @@ module.exports = {
 		return unit.symbol;
 	},
 
-	formatBytesWithUnit (num, unit) {
+	formatBytesWithUnit (num, unit, delimiter = ' ') {
```

Comment:
There is no need for this parameter because there should *always* be a space, that's why I removed the `delimiter` option before.

Reply:
ok